tests/accessibility-assertions.js

```diff
@@ -38,7 +38,7 @@ function assertActiveElement(element) {
       return;
     }
 
-    Qunit.assert.equal(element, document.activeElement);
+    Qunit.assert.equal(document.activeElement, element);
```

Comment:
It looks bizarre in case of error, as the second argument is 'expected'  and the first 'actual'.

Reply:
What do you mean in the case of error? You mean when this assertion fails? Can you give me an example.

Maybe I'm missing something because `Qunit.assert.equal(element, document.activeElement);` reads correctly to me in that I want to assert that this particular element (actual) is indeed the active element (which we know is the document.activeElement - expected).

That seems accurate to me. What am I missing @far-fetched ?

Reply:
When test fails, with previous version: `Qunit.assert.equal(element, document.activeElement);`:
<img width="800" alt="Captura de pantalla 2021-06-11 a las 10 47 43" src="https://user-images.githubusercontent.com/11621383/121659521-aa8fde00-caa2-11eb-809a-111280832f1f.png">

But we expect that button will be focused.
So after this change:
 
<img width="773" alt="Captura de pantalla 2021-06-11 a las 10 53 01" src="https://user-images.githubusercontent.com/11621383/121660086-3bff5000-caa3-11eb-8890-9d928ea8b54e.png">

For me, element passed to this helper is not actual (we rather expect that passed element is actual) - Actual focused element is - `document.activeElement` so it should be first passed to `equal`. Does it make sense to you ?

Reply:
👍